packages/drivers/odsp-driver/src/odspDelayLoadedDeltaStream.ts

```diff
@@ -81,6 +93,7 @@ export class OdspDelayLoadedDeltaStream {
 		private readonly hostPolicy: HostStoragePolicy,
 		private readonly epochTracker: EpochTracker,
 		private readonly opsReceived: (ops: ISequencedDocumentMessage[]) => void,
+		private readonly docService: OdspDocumentService,
```

Comment:
I'd erase the type - it seems weird that connection object needs to depend on service object.
Looking through the code, this could be converted to callback / delegate that caller provides, and this class does not care where it comes from - just calls that callback when appropriate.